nemo/collections/asr/parts/jasper.py

```diff
@@ -315,6 +315,68 @@ def forward(self, x):
 
 
 class JasperBlock(nn.Module):
+    """
+    Constructs a single "Jasper" block. With modified parameters, also constructs other blocks for models
+    such as `QuartzNet` and `Citrinet`.
+
+    - For `Jasper`    : `separable` flag should be False
+    - For `QuartzNet` : `separable` flag should be True
+    - For `Citrinet`  : `separable` flag and `se` flag should be True
+
+    Note that above are general distinctions, each model has intricate differences that expand over
+    multiple such blocks.
```

Comment:
Should we point users to the models/config documentation in a sort of "see more details here"?

Reply:
Good point, will say - for further details on differences, please review the encoder Config definitions used by asr example scripts